Test/lit.site.cfg

```diff
@@ -207,16 +207,17 @@ elif "18.04" in ver and "Ubuntu" in ver:
 else:
     config.available_features = [ 'ubuntu', 'posix' ]
 
-# Sanity check: Check solver executable is available
-solverRoots = os.pathsep.join(
-    (path.dirname(dafnyExecutable), binaryDir, os.environ["PATH"])
-)
-
-print(solverRoots)
+def find(name, rooot):
+    for root, dirs, files in os.walk(rooot):
+        for file in files:
+            if file.endswith(name):
+                return name
+    return ""
 
 solverPath = \
-    lit.util.which("z3-4.8.5", solverRoots) or \
-    lit.util.which("cvc4", solverRoots)
+    find("z3-4.8.5", binaryDir) or \
+    find("z3-4.12.1", binaryDir) or \
```

Comment:
It seems reasonable to include both 4.8.5 and 4.12.1 in the list of alternatives, but were you having trouble where not having 4.12.1 was causing failures? On `master`, 4.8.5 should still be the default, and both versions should be available on all current branches at the moment. I'm wondering whether there's something deeper going wrong that we should fix.